includes/Core/Authentication/Authentication.php

```diff
@@ -1054,12 +1054,14 @@ private function get_reconnect_after_url_mismatch_notice() {
 				'content'         => function() {
 					$connected_url = $this->connected_proxy_url->get();
 					$current_url   = $this->context->get_canonical_home_url();
-					$content       = sprintf(
-						'<p>%s <a href="%s">%s</a></p>',
-						esc_html__( 'Site Kit by Google: Looks like the URL of your site has changed. In order to continue using Site Kit, you’ll need to reconnect, so that your plugin settings are updated with the new URL.', 'google-site-kit' ),
+					$content       = '<p>' . sprintf(
```

Comment:
I moved the wrapping `<p>` tag outside the translation as PHPCS complained about wrapping HTML tags for the translation string.

Reply:
That makes sense, I don't think the `p` tag needs to be translatable either 👍

Reply:
Thank you!